README.md

```diff
@@ -565,31 +579,113 @@ json schema declared.
             "Content-Type": "application/json"
         }
     },
-    "response": {
-        "status": 201,
-        "headers": {
-            "Content-Type": "application/json"
+    "responses": [
+        {
+          "status": 201,
+          "headers": {
+              "Content-Type": "application/json"
+          }
         }
-    }
+    ]
   },
   {
       "request": {
           "method": "POST",
           "endpoint": "/gophers"
       },
-      "response": {
-          "status": 400,
+      "responses": [
+          {
+            "status": 400,
+            "headers": {
+                "Content-Type": "application/json"
+            },
+            "body": "{\"errors\":\"bad request\"}"
+          }
+      ]
+  }
+]
+````
+So as you can see, we have first of all the imposters with the restrictive, `headers` and `json schema`, but
+our last `imposter` is a simple `imposter` that it will match with any call via POST to `/gophers`.
+
+### Create an Imposter with repeated responses
+
+Killgrave allow repeatable/random responses, with this feature we can use one endpoint and obtain repeatable responses based on the settings provided by the user in imposter config.
+
+There are totally two new fields which needs to be configured for this feature.
+1. `request` -> `responseMode`
+    - `RANDOM` and `BURST` are valid values 
+    - `RANDOM` is default if field is not provided or any other value is provided.
+2. `response` -> `burst` (applicable only in `BURST` mode, ignored in)
+    - +ve integer are valid values.
```

Comment:
What do you mean by `+ve`? 🤔

Reply:
Here, +ve means positive. I'll change it to be positive instead +ve